tox.ini

```diff
@@ -15,7 +15,7 @@ extras =
     testing
 deps =
     cairo-lang==0.10.0
-    starknet-devnet==0.3.0
+    starknet-devnet==0.3.1
```

Comment:
I'm thinking we should probably set `starknet-devnet` in the `setup.cfg` reqs as well. There's already an issue for tackling dep management (#134). As it stands now, I believe users have to manually install the devnet. Just leaving this as a reminder

Reply:
Right now `nile init` installs devnet (without specifying version), potentially bringing issues. For example 0.3.0 predeployed accounts doesn't work with the current Account interface, totally agree with you, we should handle these requirements.